packages/react/src/Link.ts

```diff
@@ -67,10 +67,10 @@ const Link = forwardRef<unknown, InertiaLinkProps>(
     ref,
   ) => {
     const visit = useCallback(
-      (event) => {
+      (event: React.MouseEvent) => {
```

Comment:
Shouldn't this be a mouse _or_ keyboard event? Tabbing onto a link/button and pressing "enter" can trigger a click

Reply:
I lied. [It's still a mouse event.](https://github.com/facebook/react/issues/3907) Carry on 😅